lib/prepare.js

```diff
@@ -236,7 +236,7 @@ async function genComponentRegistrationFile ({ sourceDir }) {
   return `import Vue from 'vue'\n` + components.map(genImport).join('\n')
 }
 
-const indexRE = /\b(index|readme)\.md$/i
+const indexRE = /(?<=(^|\/))(index|readme)\.md$/i
 const extRE = /\.(vue|md)$/
```

Comment:
`lookbehind` is good, how about `/(^|\/)(index|readme)\.md$/i`? it's same for this case.

Reply:
It's not the same. `\b` will not  match the word boundary itself. But `/(^|\/)(index|readme)\.md$/i` will match the begining `/`

Reply:
I meant  /(^|\/)(index|readme)\.md$/i too... /(^|\/)(index|readme)\.md$/i has different behavior

Reply:
Well, in other words, `\b` and `(?<=(^|\/))` won't include the begining `/`, but `(^|\/)` will